### PR TITLE
Add property watched to episode to_json.

### PR DESCRIPTION
**Pull request by p0psicles**
* Change content from array to object.
* Update api-description.yml

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Removing weight from #6709

### CODE REVIEW

File: dredd/api-description.yml [hunk 1]
```diff
@@ -1215,11 +1215,15 @@ definitions:
...
+        type: object
         description: Additional content
-        items:
-          type: string
-          enum: [NFO, thumbnail]
+        properties:
+          hasNfo:
+            type: boolean
+            description: Whether the episode has an .nfo file
+          hasNfo:
```
Comment (sharkykh) [inline dredd/api-description.yml]:
```suggestion
          hasTbn:
```
---
But personally, I think the keys should be named `nfo` and `thumbnail`.
```yml
      content:
        type: object
        description: Additional content
        properties:
          nfo:
            type: boolean
            description: Whether the episode has an .nfo file
          thumbnail:
            type: boolean
            description: Wheter the episode has a thumbnail file
```

File: dredd/api-description.yml [hunk 2]
```diff
@@ -1215,11 +1215,15 @@ definitions:
...
+            type: boolean
+            description: Wheter the episode has a thumbnail file
```
Comment (sharkykh) [inline dredd/api-description.yml]:
```suggestion
            description: Whether the episode has a thumbnail file
```

File: medusa/tv/episode.py [hunk 3]
```diff
@@ -1077,11 +1078,11 @@ def to_json(self, detailed=True):
         data['airDate'] = self.air_date
         data['title'] = self.name
         data['description'] = self.description
-        data['content'] = []
+        data['content'] = {}
```
Comment (sharkykh) [inline medusa/tv/episode.py]: Can you put this below, where you set the content? (line 1104)

File: dredd/api-description.yml [hunk 4]
```diff
@@ -1295,15 +1299,18 @@ definitions:
...
           type: string
-      relatedEpisodes:
+      wantedQualities:
         type: array
-        example: [s03e08, s03e09]
         items:
           type: string
+      watched:
+        type: boolean
+        description: Wheter the episode has been marked as watched
```
Comment (sharkykh) [inline dredd/api-description.yml]:
```suggestion
        description: Whether the episode has been marked as watched
```

File: medusa/tv/episode.py (hunk 3 above)
Comment (p0psicles) [reply medusa/tv/episode.py]: Sure

**Pull request merged**